docs/Deploying/Configuration.md

```diff
@@ -1071,6 +1071,12 @@ to the amount of time it takes for Fleet to give the host the label queries.
 
 Note that currently, if both the failing policies webhook *and* this `osquery.enable_async_host_processing` option are set, some failing policies webhooks could be missing (some transitions from succeeding to failing or vice-versa could happen without triggering a webhook request).
 
+It can be set to a single boolean value ("true" or "false"), which controls all async host processing tasks, or it can be set for specific async tasks using a syntax similar to an URL query string or parameters in a Data Source Name (DSN) string, e.g. "label_membership=true&policy_membership=true". The supported async task names are:
```

Comment:
Let's clarify that when using the "URL query", if the field is not set, e.g. `host_last_seen` in the case of `label_membership=true&policy_membership=true`, then it will be set the default value.